src/components/PhonicWordSelector/PhonicWordSelector.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import { Button } from "reactstrap";
+import "../WordSelector/WordSelector.scss";
+import "../PhonicIcon/PhonicIcon.scss";
+function PhonicWordSelector(props) {
+  const [selected, setSelected] = useState(false);
+  function handleSelect() {
+    //if previously selected, unselect it
+    if (selected) {
+      props.handleSelectPhonics(false, props.name, props.index);
+    } else {
+      props.handleSelectPhonics(true, props.name, props.index);
+    }
+    setSelected(!selected);
+  }
+
+  function wordDisplay(word) {
+    return <div className="Words">{word}</div>;
+  }
+  return (
+    <div className="PhonicIcon">
+      <div className="PhonicIconBorder">
+        <div className="GroupTitle">{props.name}</div>
+        <div>{props.data.map(word => word.words.map(wordDisplay))}</div>
```

Comment:
i'm not sure if you use wordDisplay another time but you can also just do one of those `(word) => return <div className="Words">{word}</div>;` i believe instead of making a function for it